<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#background{
background-color: qlineargradient(spread:pad, x1:0.323751, y1:0.188, x2:0.727, y2:0.977409, stop:0 rgba(192, 192, 192, 255), stop:1 rgba(255, 255, 255, 255));
}

QLabel#indexTitle{
font: 24pt &quot;MS Reference Sans Serif&quot;;
}

QCheckBox{
font: 16pt &quot;MS Reference Sans Serif&quot;;
}

QWidget#checkboxesContainer{
background-color: #ffffff;
border-radius: 10px;
}

QLabel#checkboxesContainerTitle{
font: 16pt &quot;MS Reference Sans Serif&quot;;
}

QPushButton#startButton{
border-radius:5px;
font: 14pt &quot;MS Reference Sans Serif&quot;;
background-color:rgb(205, 124, 255);
}

QLabel#warning{
color : #FF0000;
font: 6.5pt &quot;MS Reference Sans Serif&quot;;
}</string>
    </property>
    <widget class="QLabel" name="indexTitle">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>90</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Hiragana and Katakana Trainer</string>
     </property>
    </widget>
    <widget class="QWidget" name="checkboxesContainer" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>961</width>
       <height>331</height>
      </rect>
     </property>
     <widget class="QCheckBox" name="Hiragana">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Hiragana</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Katakana">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Katakana</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="checkboxesContainerTitle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Select the syllabary(s) you want to practice: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>211</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="warning">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>187</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Options can't be made unchecked!!!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
